metadata-comparison/src/main/java/org/codice/nitf/metadatacomparison/FileComparer.java

```diff
@@ -97,15 +103,14 @@ private void generateOurMetadata() {
     }
 
     private void outputData() {
-        try {
-            FileWriter fstream = new FileWriter(filename + OUR_OUTPUT_EXTENSION);
+        try (FileWriter fstream = new FileWriter(filename + OUR_OUTPUT_EXTENSION)) {
```

Comment:
This try with resources should be a safe change to address a sonarlint finding because the stream is wrapped in the BufferedWriter which is closed on line 122/127 in this method.

Reply:
This is test code anyway.